src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -30,6 +30,18 @@ static void Main()
                 //         int x = default;
                 Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion7, "default").WithArguments("default literal", "7.1").WithLocation(6, 17)
                 );
+
+            var tree = comp.SyntaxTrees.First();
+            var model = comp.GetSemanticModel(tree);
+            var nodes = tree.GetCompilationUnitRoot().DescendantNodes();
+
+            var def = nodes.OfType<LiteralExpressionSyntax>().Single();
+            Assert.Equal("System.Int32", model.GetTypeInfo(def).Type.ToTestDisplayString());
```

Comment:
>Type [](start = 64, length = 4)

When does .Type differ from .ConvertedType?

Reply:
Earlier in this PR, I had the `.Type` reflect the natural type (ie. could be null) and `.ConvertedType` reflect the final type (potentially target-typed). But this has negative effects on consumers like the IDE, so I'm keeping the public API as-is, even though our bound nodes now have correct information.
Because of that .Type and .ConvertedType are always the same for `default` literals.

---
In reply to: [314092422](https://github.com/dotnet/roslyn/pull/37596#discussion_r314092422) [](ancestors = 314092422)